tests/sentry/test_integrations.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+
+from sentry import integrations
+from sentry.integrations.vsts_extension import VstsExtensionIntegrationProvider
+from sentry.testutils import TestCase
+
+
+class TestIntegrations(TestCase):
+    def test_excludes_non_visible_integrations(self):
+        # The VSTSExtension is not visible
+        assert all(
+            not isinstance(i, VstsExtensionIntegrationProvider)
+            for i in integrations.all()
+        )
```

Comment:
Is this the right place for this test file to live? can it live under `tests/sentry/integrations/`

Reply:
Yea, maybe. I was just mimicking the top level path, but I guess this should actually be in a `TestManager` spec now that I think about it.

Reply:
Moves this into `tests/sentry/integrations/test_manager.py`